checks/readme.php

```diff
@@ -69,6 +69,33 @@ function check_for_warnings() {
 		}
 	}
 
+	public function check_tested_up_to() {
+		$request = wp_remote_get( 'https://api.wordpress.org/core/version-check/1.7/' );
+
+		if ( is_wp_error( $request ) ) {
+			return;
+		}
+
+		$api_response   = json_decode( wp_remote_retrieve_body( $request ), true );
+		$latest_release = $api_response['offers'][0]['version'];
```

Comment:
```suggestion
		if ( defined( 'WP_CORE_STABLE_BRANCH' ) && WP_CORE_STABLE_BRANCH ) {
			$latest_branch = WP_CORE_STABLE_BRANCH;
		} else {
			$request            = wp_remote_get( 'https://api.wordpress.org/core/version-check/1.7/' );
			$api_response  = json_decode( wp_remote_retrieve_body( $request ), true );
			// Convert from x.y.z to x.y
			$latest_branch = (string)(float) $api_response['offers'][0]['version'] ?? false;
		}
```

There's a constant on WordPress.org we can take advantage of here, to skip over a remote request.

Ideally this would also use a transient cache or something around the HTTP request, but I'm not fussed about that. That's also why I'm not suggesting to use `get_site_transient( 'update_core' )` here, because we might as well avoid edge cases by fetching raw.

Reply:
Additionally, this changes from `$latest_release` to `$latest_branch`. Plugins are assumed tested with x.y.Z if it's marked as x.Y. ie. 6.2.2 is assumed if it's got 6.2 listed. No need to list x.y.z in this field.